firebase-auth/src/commonTest/kotlin/dev/gitlive/firebase/auth/auth.kt

```diff
@@ -71,6 +71,7 @@ class FirebaseAuthTest {
     }
 
     @Test
+    @Ignore
```

Comment:
Could also just delete this test. It's gonna fail cause the methods it tests are deprecated and only return an empty list